glog.yaml

```
@@ -0,0 +1,60 @@
+package:
+  name: glog
+  version: 0.6.0
+  epoch: 0
+  description: "C++ implementation of the Google logging module"
+  copyright:
+    - license: "BSD-3-Clause"
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - cmake
+      - build-base
+      - libucontext-dev
+      - libunwind-dev
+      - samurai
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/google/glog
+      tag: v${{package.version}}
+      expected-commit: b33e3bad4c46c8a6345525fd822af355e5ef9446
+
+  - uses: cmake/configure
+    with:
+      opts: |
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_INSTALL_LIBDIR=lib \
+        -DBUILD_SHARED_LIBS=ON \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DWITH_GFLAGS=ON \
+        -DWITH_THREADS=ON \
+        -DWITH_TLS=ON \
+        -DPRINT_UNSYMBOLIZED_STACK_TRACES=OFF \
+        -G Ninja
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+subpackages:
+  - name: "glog-dev"
+    description: "multi-language toolbox for accelerated data interchange and in-memory processing (development files)"
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      provides:
+        - glog-dev=${{package.full-version}}
```

```suggestion
    dependencies:
      runtime:
        - glog
```
melange will auto add the provides

---

fixed